apps/web/pages/[...slug].vue

```diff
@@ -50,24 +49,28 @@ const handleQueryUpdate = async () => {
   }
 };
 
-await handleQueryUpdate();
-
-const category = ref(productsCatalog.value.category);
+onNuxtReady(async () => {
+  await handleQueryUpdate().then(() => setCategoriesPageMeta(productsCatalog.value, getFacetsFromURL()));
```

Comment:
what is the difference? shouldn't it work the same with the await and next line?